src/main/java/net/glowstone/io/anvil/AnvilChunkIoService.java

```diff
@@ -208,27 +208,29 @@ public void write(GlowChunk chunk) throws IOException {
             CompoundTag sectionTag = new CompoundTag();
```

Comment:
If you add a individual init sections method, could you add methods for reading and writing individual sections?